mscore/scoreview.cpp

```diff
@@ -34,6 +34,7 @@
 #include "textcursor.h"
 #include "textpalette.h"
 #include "texttools.h"
+#include <qtoolbar.h>
```

Comment:
I'm able to compile on my linux without this include line.  It must be already include somewhere up in the include hierarcy, so I don't think it should be included here (at it looks out of place since every other include is a "musescorefile.h".